modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -16,45 +26,60 @@ to internal hosts that are outside the {product-title} cluster.
 - A Pod cannot reach specified internal subnets or hosts outside the {product-title} cluster.
 - A Pod can connect to only specific external hosts.
 
-You configure an egress firewall policy by creating an EgressNetworkPolicy Custom Resource (CR) object and specifying an IP address range in CIDR format or by specifying a DNS name.
 For example, you can allow one project access to a specified IP range but deny the same access to a different project. Or you can restrict application developers from updating from Python pip mirrors, and force updates to come only from approved sources.
 
+You configure an egress firewall policy by creating an {kind} custom resource (CR) object. The egress firewall matches network traffic that meets any of the following criteria:
+
+- An IP address range in CIDR format
+ifdef::openshift-sdn[]
+- A DNS name that resolves to an IP address
+endif::openshift-sdn[]
+ifdef::ovn[]
+- A port number
+- A protocol that is one of the following protocols: TCP, UDP, and SCTP
+endif::ovn[]
+
+ifdef::openshift-sdn[]
 [IMPORTANT]
 ====
-You must have OpenShift SDN configured to use either the network policy or multitenant modes to configure egress firewall policy.
+You must have OpenShift SDN configured to use either the network policy or multitenant mode to configure an egress firewall.
 
-If you use network policy mode, egress policy is compatible with only one policy per namespace and will not work with projects that share a network, such as global projects.
+If you use network policy mode, an egress firewall is compatible with only one policy per namespace and will not work with projects that share a network, such as global projects.
 ====
+endif::openshift-sdn[]
 
 [CAUTION]
 ====
-Egress firewall rules do not apply to traffic that goes through routers. Any user with permission to create a Route CR object can bypass egress network policy rules by creating a route that points to a forbidden destination.
+Egress firewall rules do not apply to traffic that goes through routers. Any user with permission to create a Route CR object can bypass egress firewall policy rules by creating a route that points to a forbidden destination.
 ====
 
 [id="limitations-of-an-egress-firewall_{context}"]
 == Limitations of an egress firewall
 
 An egress firewall has the following limitations:
 
-* No project can have more than one EgressNetworkPolicy object.
+* No project can have more than one {kind} object.
 
-* A maximum of 1 EgressNetworkPolicy object with a maximum of 50 rules can be defined per project.
+* A maximum of 1 {Kind} object with a maximum of 50 rules can be defined per project.
 
-* The `default` project cannot use egress network policy.
+* The `default` project cannot use an egress firewall.
```

Comment:
EgressFirewall will work in the default project in ovn-kubernetes but not in sdn